main/src/library/Process.flix

```diff
@@ -84,11 +99,38 @@ mod Process {
                     k(ProcessHandle.ProcessHandle(process))
                 } catch {
                     case ex: IllegalArgumentException  => Err(IoError(ErrorKind.InvalidInput, ex.getMessage()))
-                    case ex: NullPointerException      => Err(IoError(ErrorKind.InvalidInput, ex.getMessage()))
-                    case ex: SecurityException         => Err(IoError(ErrorKind.PermissionDenied, ex.getMessage()))
                     case ex: IOException               => Err(IoError(ErrorKind.Other, ex.getMessage()))
                 }
             }
+
+            def exitValue(ph, k) =
+                try {
+                    match ph {
```

Comment:
Let us rewrite all the match ph to simple let bindings, let ProcessHandle.ProcessHandle(jproc). Maybe also use ProcessHandle so its shorter.